themes/godotengine/pages/features.htm

```diff
@@ -56,18 +58,18 @@ <h2>Innovative design</h2>
 		</p>
 	  </div>
 	</div>
-  </section>
+</section>
 
-  <section id="features_3d" class="container sm-full">
+<section id="features_3d" class="container sm-full">
 	<a href="{{ 'assets/features/3dgames.jpg' | theme }}" class="lightbox" data-group="features">
 	  <img
-		src="{{ 'assets/features/3dgames.jpg' | theme }}"
```

Comment:
It seems like you are changing from tabs to spaces for indenting in some areas. I don't have any strong preference for either, but I'm not sure if the project has some indentation rules set up by the linter. Maybe @Calinou or @YuriSizov can clear it up?

Reply:
oops, fixed that